modules/trusted-firmware-m/Kconfig

```diff
@@ -151,7 +151,7 @@ config TFM_ISOLATION_LEVEL
 
 config TFM_BL2
 	bool "Add MCUboot to TFM"
-	default y
+	default n # Does not currently work
```

Comment:
If this does not currently work, then I suggest removing the prompt so that users don't enable this by mistake.

Reply:
You will be able to use TF-M without nrf_security, so I don't think promptless is good

Maybe add a depends for !NORDIC_SECURITY_BACKEND to remove the prompt?

Reply:
and make the message more precise on when / why this is not working, and when it is working.